EfEnumToLookupTests/EfEnumToLookupTests.csproj

```diff
@@ -100,6 +100,9 @@
       <Name>EfEnumToLookup</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
Typical, lol. I'm still on 2013 for the moment. I'd take this out until we at least know what it does.